### ISSUE TITLE
Train MagicPoint

### ISSUE THREAD

**Issue by shaofengzeng** [jeethesh-pai/Superpoint_training#2]
Hi, can you share the final training loss about magicpoint on synthetic and coco data sets ? and how many (average number) keypoints will the magicpoint detected on coco data set?

**Comment by jeethesh-pai** [comment]
Hi, 

Here are the losses for my training. I trained for around 400k iterations. You can try for further more but as per the paper in order to avoid overfitting to such a simple dataset, I stopped it.

![grafik](https://user-images.githubusercontent.com/75034628/145476482-448f1f58-bda1-49e2-9a33-138fca6fec36.png)
![grafik](https://user-images.githubusercontent.com/75034628/145476523-68e070c4-ded5-4eb6-9a2b-5f6c740277fe.png)

Also, the number of keypoints detected on coco datasets depends on the confidence threshold. Supposing you set a very small threshold you get more number of keypoints. Also you can control the no. of keypoints being detected using the parameter top_k in the configuration file.
